code/modules/reagents/chemistry/reagents/impure_reagents/impure_toxin_reagents.dm

```diff
@@ -30,7 +30,8 @@
 
 /datum/reagent/impurity/methanol/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
 	var/obj/item/organ/internal/eyes/eyes = affected_mob.get_organ_slot(ORGAN_SLOT_EYES)
-	eyes?.apply_organ_damage(0.5 * REM * seconds_per_tick, required_organ_flag = affected_organ_flags)
+	if(eyes && !IS_ROBOTIC_ORGAN(eyes))
+		eyes.apply_organ_damage(0.5 * REM * seconds_per_tick, required_organ_flag = affected_organ_flags)
```

Comment:
This one already passes `affected_organ_flags`? So it shouldn't need any changes.

Reply:
Applied with https://github.com/tgstation/tgstation/pull/76766/commits/7c68d2e99d21ee1df52b4202275d9d8fb2ff4a8f  seems all chems that damage or heal organs already have the `ORGAN_ORGANIC` flag set.  I need to code dive and see if 
 robotic liver is completely immune to damage from toxins.